### PR TITLE
Create table images and musican skils

### PR DESCRIPTION
**Pull request by YanPam**
)

### CODE REVIEW

File: app/models/musician_skills_user.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class MusicianSkillsUser < ApplicationRecord
+  belongs_to :musician_skill
+  belongs_to :users
```
Comment (andreydro) [inline app/models/musician_skills_user.rb]: `belongs_to :user`

File: db/migrate/20190807074256_create_musician_skills_users.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#:reek:FeatureEnvy and :reek:UncommunicativeVariableName
+class CreateMusicianSkillsUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :musician_skills_users do |t|
```
Comment (andreydro) [inline db/migrate/20190807074256_create_musician_skills_users.rb]: This is connection table and I guess you need `user_id` and `musician_skill_id` in here

Comment (andreydro) [reply db/migrate/20190807074256_create_musician_skills_users.rb]: What for do you need `state` in here?

**Comment by longbeardmen** [comment]
https://github.com/rubizza-camp/uber-music/pull/2#issuecomment-519041932

**Pull request closed without merging**